tests/validation/cattlevalidationtest/core/test_lb.py

```diff
@@ -971,6 +971,8 @@ def test_lb_with_health_check_without_uri(client):
     delete_all(client, [lb, con1])
 
 
+@pytest.mark.skipif(
+    True, reason='round robin access check does not always apply')
```

Comment:
I was thinking at some point , i will modify this test case to log into LB Agent and make sure that haproxy config is set to leastconn . That is why this test case is left in "skipped" state.